chapters/002-smol-internet/README.md

```diff
@@ -190,251 +149,205 @@ Let's go back to our `docker-compose.yml` and give `boudi` an additional network
+
 ```bash
-root@tara:/# tcpdump -ne
-18:29:45.268130 02:42:0a:01:02:02 > 02:42:a9:f7:9e:4f, ethertype IPv4 (0x0800), length 98: 10.1.2.2 > 10.1.1.3: ICMP echo request, id 30, seq 1, length 64
-18:29:50.570491 02:42:0a:01:02:02 > 02:42:a9:f7:9e:4f, ethertype ARP (0x0806), length 42: Request who-has 10.1.2.1 tell 10.1.2.2, length 28
-18:29:50.570543 02:42:a9:f7:9e:4f > 02:42:0a:01:02:02, ethertype ARP (0x0806), length 42: Reply 10.1.2.1 is-at 02:42:a9:f7:9e:4f, length 28
+18:53:54.167168 ARP, Reply 10.1.2.3 is-at 02:42:0a:01:02:03, length 28
 ```
 
-From this `tcpdump`, we can see `tara`'s mac address, `02:42:0a:01:02:02` attempting to reach `02:42:a9:f7:9e:4f`. But remember, `boudi`'s mac address is `02:42:0a:01:01:03`. This tells us that `tara` is making a hail mary to reach `10.1.1.3` via the default gateway that docker configured for us automatically.
+Now back to our regularly scheduled exploration!
+
+### Can `tara` ping `boudi` on the `squasheeba` network?
 
-But wait! That's not the behavior we want... Let's get rid of that default gateway. To do it manually, we can `hopon tara` and run
+We've already seen that `tara` can respond to `boudi`'s `ping`s that were issued from `boudi`'s interface on `doggonet`. We can double check that `tara` can initiate the `ping`, just for fun:
 
 ```bash
-root@tara:/# ip route del default
+root@tara:/# ping 10.1.2.3 -c 2
+PING 10.1.2.3 (10.1.2.3) 56(84) bytes of data.
+64 bytes from 10.1.2.3: icmp_seq=1 ttl=64 time=0.245 ms
+64 bytes from 10.1.2.3: icmp_seq=2 ttl=64 time=0.242 ms
+
+--- 10.1.2.3 ping statistics ---
+2 packets transmitted, 2 received, 0% packet loss, time 1041ms
+rtt min/avg/max/mdev = 0.242/0.243/0.245/0.001 ms
 ```
 
-Now, when we try to ping `boudi` on the `squasheeba` network, we get the failure we expect:
+Now... let's see if `tara` is able to ping `boudi`'s interface on the `squasheeba` network...
 
 ```bash
 root@tara:/# ping 10.1.1.3
 ping: connect: Network is unreachable
 ```
 
-And, watching carefully, we see that the `tcpdump` on `tara` no longer has any output.
+Lovely! This is because, while `boudi` has an interface on the `doggonet` network and `tara` and `boudi` can directly communicate on that network, `tara` doesn't know anything about the `squasheeba` network; not even that `boudi` knows about it. We need to define a route for `tara` that tells it how to get to the `squasheeba` network!
 
-BOOM! Good job, team.
+### Make `tara` ping hosts on the `squasheeba` network
 
-For the future, we can make this the default setup by changing the  [start-up script](./init/start-up.sh) to look like:
+The first thing we need to do is add a route from `tara` to the `squasheeba` network via `boudi`. Because `boudi` has routes to both `doggonet` and `squasheeba`, `boudi` can act as the gateway between the two. We can manage routes on our machines using the `ip route` command:
```

Comment:
```suggestion
The first thing we need to do is add a route from `tara` to the `squasheeba` network via `boudi`. Because `boudi` has routes to both `doggonet` and `squasheeba`, `boudi` can act as the router between the two. We can manage routes on our machines using the `ip route` command:
```
I think we're preferring the term `router` instead of the synonym `gateway`